Havit.Blazor.Components.Web.Bootstrap/Grids/HxGrid.razor

```diff
@@ -104,6 +105,24 @@
 				</tr>
 			</thead>
 			<tbody>
+                @{
+                    RenderFragment<int> placeholderRowTemplate = (int rowIndex) =>
+                    @<tr @key="@rowIndex"
+                                height="@((ContentNavigationModeEffective == GridContentNavigationMode.InfiniteScroll) ? rowHeight : null)"
+                                class="@ItemRowCssClassEffective"
+                            @onclick:stopPropagation>
+
+                            @foreach (IHxGridColumn<TItem> column in columnsToRender)
+                            {
+                                // same as above this else section
+                                GridCellTemplate cellTemplate = column.GetItemPlaceholderCellTemplate(new GridPlaceholderCellContext { Index = rowIndex });
```

Comment:
We should not need to create a new `GridPlaceholderCellContext` instance for every single iteration of the loop. For now, same instance can be used for all columns (as it does not hold any column-specific context yet).